### PR TITLE
DNN-20316: Cannot create new user with email address as username

### PR DESCRIPTION
**Pull request by Mohtshum**
- Pull string to static field to be used in other project as well

### CODE REVIEW

**Comment by mitchelsellers** [comment]
When looking at the space checking why not use string.IsNullOrWhiteSpace() to check since it captures more options of whitespace?

**Comment by Mohtshum** [comment]
Please review changes

File: DNN Platform/Library/Entities/Users/IUserController.cs [hunk 1]
```diff
@@ -107,5 +107,12 @@ public interface IUserController
         /// in background threads or scheduler jobs. 
         /// </remarks>
         string GetUserProfilePictureUrl(int portalId, int userId, int width, int height);
+
+        /// <summary>
+        /// Check username against
```
Comment (tpluscode) [inline DNN Platform/Library/Entities/Users/IUserController.cs]: Nit: Please complete the comment ✍️

Comment (Mohtshum) [reply DNN Platform/Library/Entities/Users/IUserController.cs]: Thnx, too fast too furious, how could I missed that 😮

**Comment by Mohtshum** [comment]
Changes pushed

**Comment by tpluscode** [comment]
Looking good otherwise @mitchelsellers ?

**Pull request merged**